src/shared/components/palette/case-link/components/read-linked-cases.component.ts

```diff
@@ -68,9 +68,8 @@ export class ReadLinkedCasesComponent implements OnInit, AfterViewInit {
   }
 
   public reloadCurrentRoute() {
-    let currentUrl = this.router.url;
-    this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
-      this.router.navigate([currentUrl]);
+    this.router.navigate(['cases', 'case-details', this.linkedCasesService.caseDetails.case_id]).then(() => {
+      window.location.hash = 'Linked cases';
```

Comment:
1. suppose it shouldn't use blank string in the hash? 
2. and does it have that hash key in the html template?

Reply:
As far as I know, there is no hash key for the linked  cases and we usually redirect to the landing page tab.

Reply:
so does that mean this window.location.hash never work can we remove it?